testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/Multinode.java

```diff
@@ -46,46 +38,26 @@
     private final DockerFiles dockerFiles;
     private final DockerFiles.ResourceProvider configDir;
 
-    private final String imagesVersion;
-    private final File serverPackage;
-    private final boolean debug;
-
     @Inject
     public Multinode(
             DockerFiles dockerFiles,
-            Standard standard,
-            Hadoop hadoop,
-            EnvironmentConfig environmentConfig,
-            @ServerPackage File serverPackage,
-            @Debug boolean debug)
+            TrinoMultinode standardMultinode,
```

Comment:
A bit inconsistent. Which is the correct name? `TrinoMultinode` or `standardMultinode`?